src/test/java/sirius/kernel/SiriusExtension.java

```diff
@@ -30,4 +32,9 @@ public void beforeAll(ExtensionContext context) {
     public void beforeEach(ExtensionContext context) throws Exception {
         CallContext.initialize();
     }
+
+    @Override
+    public void afterAll(ExtensionContext context) throws Exception {
+        TestHelper.tearDown(SiriusExtension.class);
```

Comment:
Framework starts 3 times. I suppose shutting it down here is wrong. What do we need that for?
https://drone.scireum.com/scireum/sirius-kernel/2905/1/2

Reply:
Also, is there no surefire collecting test statistics?

Reply:
Thanks for this improvement.

We still have to start the framework once for the Groovy Specs run via Scenario Runner, and a second time for all JUnit 5 Tests though. Instead of optimizing this further, we should try and get rid of the Groovy specs completely.

Test statistics are now also printed